src/explore-education-statistics-admin/src/pages/release/pre-release/__tests__/PreReleasePageContainer.test.tsx

```diff
@@ -26,7 +26,7 @@ describe('PreReleasePageContainer', () => {
   const testPreReleaseSummary: PreReleaseSummary = {
     publicationTitle: 'Test publication',
     publicationSlug: 'test-publication',
-    releaseTitle: 'Calendar Year 2018',
+    releaseTitle: 'Academic year 2018',
```

Comment:
Is this the correct title for an academic year?

Reply:
Doesn't really make a difference to the test suite, but I've changed this back anyway